packages/web/src/components/nav/desktop/NavColumn.js

```diff
@@ -556,6 +557,16 @@ const mapDispatchToProps = dispatch => ({
   showVisualizer: () => dispatch(openVisualizer())
 })
 
-export default withRouter(
+const ConnectedNavColumn = withRouter(
   connect(makeMapStateToProps, mapDispatchToProps)(NavColumn)
 )
+
+ConnectedNavColumn.propTypes = {
```

Comment:
Why add propTypes here? Just to get some runtime type safety until we fully type this file?